apis/beacon/states/validator_identities.yaml

```
@@ -0,0 +1,69 @@
+post:
+  operationId: "postStateValidatorIdentities"
+  summary: "Get validator identities from state"
+  description: |
+    Returns filterable list of validators identities.
+
+    Identities will be returned for all indices or public keys that match known validators. If an index or public key does not
+    match any known validator, no identity will be returned but this will not cause an error. There are no guarantees for the
+    returned data in terms of ordering.
+
+     Depending on `Accept` header data can be returned either as JSON or as bytes serialized by SSZ.
+  tags:
+    - Beacon
+  parameters:
+    - name: state_id
+      in: path
+      $ref: '../../../beacon-node-oapi.yaml#/components/parameters/StateId'
+  requestBody:
+    description: "An array of either hex encoded public keys (any bytes48 with 0x prefix) or validator indices"
```

This would make it slightly clearer that it can be a mixed array (from https://github.com/ethereum/beacon-APIs/pull/453)

```suggestion
    description: "An array of values, with each value either a hex encoded public key (any bytes48 with 0x prefix) or a validator index."
```